flycs_sdk/pipelines.py

```diff
@@ -128,9 +130,7 @@ def to_dict(self) -> Dict:
             "name": self.name,
             "version": self.version,
             "schedule": schedule,
-            "start_time": _format_datetime(self.start_time)
-            if self.start_time
-            else None,
+            "start_time": f"{self.start_time}" if self.start_time else None,
```

Comment:
Why don't we need to use the `_format_datetime` function anymore here ?
Since we should be backwards compatibles I don't see how we can get rid of it.